### PR TITLE
Update *application deployment to an env* section

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,8 +110,15 @@ Coming soon
 ### Deploy one kubernetes dockerized applications to an environment (aws)
 1. Make sure you have followed the instructions above to create or access an existing eks cluster
 2. Change the branch or tag in `config/environment_ENVNAME` if needed where ENVNAME is the environment you are deploying to.
-3. `cd apps`
-4. `make deploy-app-APPNAME` where APPNAME is the name of the ingest application. For example, `make deploy-app-ingest-core`
+4. Navigate to the root folder in your local ingest-kube-deployment repo
+5. Execute: `source config/environment_ENVNAME`
```
Comment (prabh-t) [inline README.md:114]: I suggest we add the `source config/..` in the Makefile `set-context` goal. This way we don't have to do it each time we run a make command.

Comment (ke4) [reply README.md:114]: Yes, that would be better and less error-prone. Totally agree. Then we could remove step 3 - 7

**Comment by lauraclarke** [comment]
@ke4 Is there a reason this hasn't been merged yet?

**Comment by ke4** [comment]
Old stale PR. I am closing this.

**Pull request closed without merging**